core/src/main/scala/kafka/zk/KafkaZkClient.scala

```
@@ -48,6 +48,29 @@ import scala.collection.mutable.ArrayBuffer
 class KafkaZkClient(zooKeeperClient: ZooKeeperClient, isSecure: Boolean) extends Logging {
   import KafkaZkClient._
 
+  def createSequentialPersistentPath(path: String, data: String = ""): String = {
+    val createRequest = CreateRequest(path, data.getBytes("UTF-8"), acls(path), CreateMode.PERSISTENT_SEQUENTIAL)
+    val createResponse = retryRequestUntilConnected(createRequest)
+    createResponse.path
```

We should throw an exception if the result code is not OK.